pkg/scaling/scale_handler.go

```
@@ -265,11 +256,12 @@
 	h.logger.Info("Scaling up from zero", zap.String("kind", targetKind), zap.String("namespacedName", namespacedName.String()), zap.Int32("replicas", replicas))
 
 	var err error
+	var scaled bool = false
```

golangci-lint (1.64.8)

[warning] var-declaration: should omit type bool from declaration of var scaled; it will be inferred from the right-hand side

(revive)